src/@types/assets/movie.d.ts

```diff
@@ -0,0 +1,16 @@
+type genre = {
```

Comment:
```suggestion
type Genre = {
```

`types` and `interfaces` should always be capital case (e.g., `AwniGharbia`)

Reply:
Is `AwniGharbia` a type?

Reply:
Fixed.